index.d.ts

```diff
@@ -82,8 +82,8 @@ declare namespace websocketPlugin {
   }
 
   export interface PluginOptions {
-    handle: (connection: SocketStream) => void;
-    options: WebSocket.ServerOptions;
+    handle?: (connection: SocketStream) => void;
```

Comment:
Why were these changed to optional? Sorry I'm not that familiar with the repo.

Reply:
They are optional. If not changed, type checking will report errors.

Reply:
Okay! Then LGTM